openstf/model/confidence_interval_applicator.py

```diff
@@ -7,19 +7,29 @@
 import numpy as np
 import pandas as pd
 from scipy import stats
+from sklearn.base import RegressorMixin
+
+from openstf.enums import MLModelType
 
 MINIMAL_RESOLUTION: int = 15  # Minimal time resolution in minutes
 
 
 class ConfidenceIntervalApplicator:
-    def __init__(self, model):
-        self.confidence_interval = model.confidence_interval
+    def __init__(self, model: RegressorMixin, forecast_input_data: pd.DataFrame):
+        self.model = model
+        self.forecast_input_data = forecast_input_data
 
-    def add_confidence_interval(
-        self, forecast: pd.DataFrame, quantiles: list
-    ) -> pd.DataFrame:
+    def add_confidence_interval(self, forecast: pd.DataFrame, pj: dict) -> pd.DataFrame:
         temp_forecast = self._add_standard_deviation_to_forecast(forecast)
-        return self._add_quantiles_to_forecast(temp_forecast, quantiles)
+
+        if pj["model"] == MLModelType.XGB_QUANTILE:
+            return self._add_quantiles_to_forecast_quantile_regression(
+                temp_forecast, pj["quantiles"]
+            )
+        else:
```

Comment:
Its a bit personal preference. But the `else` case is not required here since you have already returned from the `if`.